hs_core/testing.py

```diff
@@ -559,7 +559,7 @@ def timeseries_metadata_extraction(self):
         self.assertEqual(parser.parse(temporal_coverage.value['start']).date(),
                          parser.parse('01/01/2008').date())
         self.assertEqual(parser.parse(temporal_coverage.value['end']).date(),
-                         parser.parse('01/31/2008').date())
+                         parser.parse('01/30/2008').date())
```

Comment:
why was this updated?

Reply:
The temporal coverage data was used to get extracted from a different table of the sqlite file. The issue #2771 suggests to get the same data now from a different table.

Reply:
👍